### DISCUSSION TITLE
post/authentik-tutorial-1-introduction-and-install/

### DISCUSSION THREAD

**Discussion by giscus** [ecwu/ecwu.github.io#15] [Blog Post Discussion]
# post/authentik-tutorial-1-introduction-and-install/

Homepage and Blog of ECWU

https://ecwuuuuu.com/post/authentik-tutorial-1-introduction-and-install/

**Comment by WowYiJiu** [comment]
![Brand issue](https://im.wowyijiu.com/file/0a1b7b3ecf3a474210444.png)
博主您好，这个您遇到过没，我的用media下的图片就会这样

**Reply by ecwu** [reply]
会有这个情况，包括如果设定了自定义背景也会闪一下默认的。

我自己到不是很在意。出现的原因应该是官方的实现，如果有自定义徽标/背景是页面开始加载后再覆盖上去的。就会导致 Authentik 的 Logo 出现一下又变成自定义徽标。

这种情况如果要修复好，应该要把 Authentik 的 Logo 文件给覆盖掉，我看了下位置是 static/dist/assets/icons/icon_left_brand.svg

**Comment by WowYiJiu** [comment]
现在的server和work容器内目录好像也没有static目录，我尝试删除或者替换/web/dist/assets/images下的icon_left_brand.svg都不行

**Reply by ecwu** [reply]
好像是在 server 容器内 /web/dist/assets/icons

**Reply by WowYiJiu** [reply]
我server和docker容器内全都尝试了，都不行，很奇怪，这个看来只能看源码分析了。。。我抽空找一下看看能不能碰碰运气

**Reply by ecwu** [reply]
看看是不是和浏览器缓存有关，强制刷新下。

**Reply by WowYiJiu** [reply]
> 看看是不是和浏览器缓存有关，强制刷新下。
应该是缓存问题，这几天出去玩了，刚到家我打开看了下已经好了，非常感谢
